catalog/transformer/catalog-transformer-common/src/main/java/ddf/catalog/transformer/common/tika/MetacardCreator.java

```diff
@@ -46,6 +60,31 @@
 
   public static final String DURATION_METDATA_KEY = "xmpDM:duration";
 
+  public static final String ALTERNATE_MAPPING_OVERRIDE_FILE_KEY = "tika.metadata.mapping.file";
+
+  static final Map<String, List<String>> ALTERNATE_METADATA_KEY_MAPPING;
+
+  private static final Pattern DURATION_SECONDS = Pattern.compile("([+-]?\\s*\\d*\\.?\\d+)\\s*s");
+
+  private static final Pattern DURATION_HMS = Pattern.compile("(\\d+):(\\d+):(\\d*\\.?\\d+).*");
+
+  static {
+    ALTERNATE_METADATA_KEY_MAPPING = new HashMap<>();
+
+    try (InputStream mappingStream = getAlternateMappings()) {
```

Comment:
❓ Does restarting the bundle cause a new classloader to be created a reread the file? I think it will but want to make sure so we do not have to restart DDF to reread the file.

Reply:
Stopping and starting the bundle does not cause it to reload the file because that uses the same classloader.  However, if you do a `bundle:refresh`, it will use a new classloader and will reread the file.

Reply:
Good to know. I thought there was a way to destroy and create a new classloader. Glad to hear that worked. This code should be fine as it is then.